### ISSUE TITLE
Remove use of distutils upstream to unblock 3.12

### ISSUE THREAD

**Issue by timkpaine** [conda-forge/litepcie-feedstock#8]
### Solution to issue cannot be found in the documentation.

- [X] I checked the documentation.

### Issue

N/a

### Installed packages

```shell
N/a
```

### Environment info

```shell
N/a
```

**Comment by timkpaine** [comment]
I'm not going to track this anymore, open another separate issue.

https://github.com/conda-forge/pandas-datareader-feedstock/pull/22#issuecomment-3190256901
